packages/odyssey-react/src/components/Button/Button.theme.ts

```diff
@@ -13,106 +13,97 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  ...root(theme),
+  Gutter: theme.SpaceRemS,
```

Comment:
this is another variable we can probably remove I don't think anyone will want to style the space between buttons and this style is easily avoided by putting wrapping divs around buttons in a custom layout

Reply:
Same question as `LineHeight` above. We need a default here so that Buttons are properly laid out when implemented inline. That value should be tied to one of the spacing tokens, regardless of whether or not it's themed.

Reply:
@edburyenegren-okta SASS variables work for now, in the future we can spend some time thinking through how we solve for this without SASS 🙃

Reply:
Is there any reason why we don't just refer to the design token directly?

Reply:
@arnoldsandoval-okta what do you mean by refer to the design token directly?

Reply:
@edburyenegren-okta so sounds like we can remove this `Gutter` token entirely yes? Go back to sass var? I know its not ideal but at least we won't create a public API that eventually we'll want to remove!

Reply:
> @arnoldsandoval-okta what do you mean by refer to the design token directly?

@jeffbelser-okta as I have said before, design tokens are not the same thing as theming API properties. My question is best interpreted with that understanding - why cant we just use the design tokens, as they exist in the `@okta/design-tokens` package?

Reply:
I also have trouble with exposing Gutter as a public variable. As I said in stand up having a container (ButtonGroup?) feels like a better home for the long term. I'd also vote for removing from the reducer for now.